src/Common/Dwarf.h

```diff
@@ -127,15 +127,10 @@ class Dwarf final
 
     enum class LocationInfoMode
     {
-        // Don't resolve location info.
-        DISABLED,
-        // Perform CU lookup using .debug_aranges (might be incomplete).
-        FAST,
-        // Scan all CU in .debug_info (slow!) on .debug_aranges lookup failure.
-        FULL,
-        // Scan .debug_info (super slower, use with caution) for inline functions in
-        // addition to FULL.
-        FULL_WITH_INLINE,
+        DISABLED, /// Don't resolve location info.
```

Comment:
It seems in our places we use comments on top, why here we need to place them right ?

Reply:
Multiline right-padded comments are more ugly IMO.